api/models/comment/createAndPresentComment.js

```diff
@@ -106,3 +107,10 @@ const notifySockets = (comment, post) => {
     return post.pushCommentToSockets(comment)
   }
 }
+
+const createOrUpdateConnections = (userId, existingFollowers) => comment => {
+  // Deliberately non-blocking (don't wait for promise to resolve/reject)
+  existingFollowers
+    .filter(f => f !== userId)
+    .forEach(follower => UserConnection.createOrUpdate(userId, follower, 'message'))
+}
```

Comment:
We can't ever break the promise chain; it leads to unspecified behavior. If you don't want the response to block on a slow operation, look at the way we use Queue.classMethod elsewhere in the project to push work over to [kue](https://www.npmjs.com/package/kue).